### PR TITLE
Fixed up QA fixes for search resource endpoint

### PR DESCRIPTION
**Pull request by ivanleomk**

### CODE REVIEW

File: client/src/components/QueryTable.js [hunk 1]
```diff
@@ -18,6 +18,10 @@ const QueryTable = (props) => {
 
     const AddResource = () => {
         const newResource = document.querySelector('.newresource').value
+        if (newResource === "") {
```
Comment (kwannoel) [inline client/src/components/QueryTable.js:21]: Are we doing validation twice here

File: client/src/components/QueryTable.js [hunk 2]
```diff
@@ -35,6 +39,10 @@ const QueryTable = (props) => {
 
     const submitQuery = () => {
         let finalQuery = []
+        if (currentQuery.length === 0) {
```
Comment (kwannoel) [inline client/src/components/QueryTable.js:42]: And here?

Comment (kwannoel) [reply client/src/components/QueryTable.js:42]: Let's merge in #151 then look at this again

Comment (ivanleomk) [reply client/src/components/QueryTable.js:42]: Ok I'll keep this alone for now

**Pull request merged**